README.md

```diff
@@ -93,6 +93,42 @@ Fill out [config](./config.yaml) file with your details (or leave default values
 | *PASSWORD*            | *null* **(need to be overwritten)**   | User's password
 | *INTERACTIVE_MODE*    | *True*                                | Enables interactive mode (confirmation prompts).
 
+---
+### Alexa skill configuration (*Optional*)
+
+You can configure an Alexa skill to control the Sidewalk device via voice commands. This is however optional and you can skip this step if you don't want to use Alexa with this sample app. 
+
+Install the required libraries for Alexa skill.
+
+- Linux / MacOS:
+```
+python3 -m pip install requests  -t .\ApplicationServerDeployment\lambda\alexaLibs
+python3 -m pip install 'urllib3<2'  -t .\ApplicationServerDeployment\lambda\alexaLibs --upgrade
+```
+
+- Windows:
+```
+python -m pip install requests  -t .\ApplicationServerDeployment\lambda\alexaLibs
+python -m pip install 'urllib3<2'  -t .\ApplicationServerDeployment\lambda\alexaLibs --upgrade
+```
+
+| Follow the guide below to create an Alexa skill (*Optional) |
+|-------------------------------------------------------------|
+Note: 
+- You can skip the step to create a lambda function as the script will create one for you.
+
+https://developer.amazon.com/en-US/docs/alexa/smarthome/steps-to-build-a-smart-home-skill.html
```

Comment:
```suggestion
https://developer.amazon.com/en-US/docs/alexa/smarthome/steps-to-build-a-smart-home-skill.html
The documentation outlines seven steps to build a smart home skill.
- Refer to the configuration sample below for Step 3 Configure account linking. After completing this step, follow Step 4 to begin testing the skill with the Sidewalk sample application.
<img width="1775" alt="AlexaSkillConfigurationSample" src="https://github.com/aws-samples/aws-iot-core-for-amazon-sidewalk-sample-app/assets/108501191/3d58909e-06a0-46a9-9df5-38221dbf8510">
- Once you are ready to publish your skill, proceed to complete the remaining steps from 5 to 7 for testing and submitting your skill for certification.
```
Adding this clarity for developers to be aware they can start testing right away after completing Step 3, and a config sample on Step 3 is essential to accommodate developers.

Reply:
Agreed with this change

Reply:
Agreed with this change